frontend/locales/en/common.ts

```diff
@@ -51,6 +51,8 @@ const common = {
     edit: "edit",
 
     "no edition": "no edition",
+
+    logout: "logout",
```

Comment:
The common file got some structure. I think this should be added in the user-management part.